src/components/iconTypes.js

```diff
@@ -0,0 +1,16 @@
+export const HEADER = 'F';
```

Comment:
What does `F` stands for? (I know it maps for an icon, but it worth declaring it as u did in the `CUISINES` section)